galley/pkg/source/kube/schema/check/check.go

```diff
@@ -35,14 +36,58 @@ var (
 	pollTimeout  = time.Minute
 )
 
+var testHookGetAllResourceSchemas = kubeMeta.Types.All
```

Comment:
Can we simply plump this in from up-top, caller?

Reply:
done